canonical/ubuntu.com
# Building the distributor store

## Done

- Building the distributor store 

## QA
There are some changes with the checkout page and hooks that are used for all the shops, so all shops need to be QAed for a purchase.
- [x] QA the credential shop
- [x] QA the pro shop
- [x] QA the distributor shop

To QA the distributor shop 
1. Log in with a distributor account (Distributor account don't have access to normal shops)
2. Go to https://ubuntu-com-14054.demos.haus/pro/distributor
3. Click Initiate order
4. Complete all the steps on the /pro/distributor/shop page
5. Proceed to checkout
6. Check it goes `/account/checkout` page and all the details in the page
7. Make a purchase and check the purchase is successful

## Issue / Card

Fixes #
